staging/src/k8s.io/dynamic-resource-allocation/deviceclass/extendedresourcecache/extendedresourcecache_test.go

```diff
@@ -28,6 +29,7 @@ import (
 	"k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog/v2/ktesting"
+	_ "k8s.io/klog/v2/ktesting/init" // Add command line flags.
```

Comment:
which command line flags need this?

Reply:
"k8s.io/klog/v2/ktesting/init" adds "-testing.v" for changing the log verbosity.